src/postgres/commands/createPostgresServer/steps/PostgresServerSkuStep.ts

```diff
@@ -27,17 +25,26 @@ export class PostgresServerSkuStep extends AzureWizardPromptStep<IPostgresServer
             "selectPostgresSku",
             "Select the Postgres SKU and options."
         );
-        wizardContext.sku = (
-            await ext.ui.showQuickPick(this.getPicks(), { placeHolder })
-        ).data;
+        const pricingTiers: IAzureQuickPickItem<Sku | undefined>[] = await this.getPicks();
+        pricingTiers.push({
+            label: localize('ShowPricingCalculator', '$(link-external) Show pricing information...'),
+            onPicked: async () => {
+                await openUrl('https://aka.ms/AAcxhvm');
+            }, data: undefined
+        });
+
+        const pick: IAzureQuickPickItem<Sku | undefined> = await ext.ui.showQuickPick(pricingTiers, { placeHolder, suppressPersistence: true, enableGrouping: true });
+        if (pick.data) {
+            wizardContext.sku = pick.data;
+        }
```

Comment:
nit: this is what I usually do
```suggestion
        wizardContext.sku = (await ext.ui.showQuickPick(pricingTiers, { placeHolder, suppressPersistence: true, enableGrouping: true })).data;
```